src/CMakeLists.txt

```diff
@@ -62,17 +62,11 @@ set(Mapper_Common_SRCS
 
  settings.cpp
 
- map.cpp
- map_part.cpp
+ map/map.cpp
+ map/map_part.cpp
```

Comment:
Isn't this core? (At least once the remaining Qt Widgets dependencies got removed.)